src/FormRenderer/TemplateRenderer.php

```diff
@@ -53,10 +53,10 @@ public function render(Nette\Forms\Form $form): string
         $template->setFile(self::TEMPLATE_FILE);
 
         $latte = $template->getLatte();
-        $latte->addProvider('formRendererImports', $this->templateImports);
```

Comment:
Please, keep the provider in place to maintain BC?

Reply:
Ok, let's keep it.